src/js/user-profile/actions/index.js

```diff
@@ -72,5 +73,7 @@ export function acceptTerms(termsName) {
       }),
       () => dispatch({ type: ACCEPTING_LATEST_MHV_TERMS_FAILURE })
     );
+
+    getUserData(dispatch);
```

Comment:
@raquelromano-gov I tagged you specifically in the context of what we discussed on Slack yesterday about being able to refresh the `/user` API endpoint. Invoking this here in this action seems to do the trick, and the corresponding redux state gets updated. Any issues with invoking this?